docs/content/en/docs/intro-klt/_index.md

```diff
@@ -12,7 +12,7 @@ When you understand the toolkit, begin with the [getting started guide](../getti
 ----
 
 The Keptn Lifecycle Toolkit (KLT) implements observability
-for deployments that are implemented with standard GitOps tools
+for deployments and well-integrates with GitOps tools
```

Comment:
another phrasing could be
```suggestion
for deployments and seamlessly-integrates with GitOps tools
```

Reply:
Either it should be integrates well or seamlessly integrates but always without the dash in the middle